.github/workflows/build-lint-test.yml

```diff
@@ -45,3 +45,46 @@ jobs:
         run: cargo clippy --all-targets --all-features -- -D warnings
       - name: Run tests
         run: cargo test --verbose
+  node:
+    name: Node
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: frontend
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install dependencies
+        run: npm ci
+      - name: Check formatting
+        run: npx prettier --check .
+      - name: Lint
+        run: npm run lint
+      - name: Test
+        run: npm test
+      - name: Build
+        run: npm run build
+  playwright:
+    name: Playwright
+    needs: node
```

Comment:
At this point there is no actual dependency yet, right?

Reply:
True. My thinking is that the feedback is clearer when only the Frontend job fails, instead of both the Frontend and the Playwright job[^1]. That does come with the cost of the pipeline taking longer. To be perfectly honest, the reason I put the `needs` in there is to have the discussion if that makes sense.

[^1]: Assumption: if at least one Frontend test fails, at least one Playwright test will fail as well.[^2]
[^2]: GitHub **does** do footnotes! :raised_hands:

Reply:
I'd prefer the CI runs to be as fast as possible. But we'll also be using the backend as a dependency (https://github.com/kiesraad/abacus/issues/169#issuecomment-2247823109) so there's going to be a dependency on that too, so this frontend dependency doesn't really matter.